administrator/language/en-GB/com_guidedtours.ini

```diff
@@ -0,0 +1,48 @@
+; Joomla! Project
+; Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.
+; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
+; Note : All ini files need to be saved as UTF-8
+
+COM_GUIDEDTOURS_ERROR_TOUR_NOT_FOUND="Tour not found!"
+COM_GUIDEDTOURS_GUIDEDTOURS_PAGE_TITLE="Guided Tours - List of Tours"
+COM_GUIDEDTOURS_GUIDEDTOURS_TABLE_CAPTION="List of Tours"
+COM_GUIDEDTOURS_GUIDEDTOURS_LABEL_DESCRIPTION="Description"
+MOD_MENU_MANAGE_GUIDEDTOURS="Guided Tours"
+COM_GUIDEDTOURS_XML_DESCRIPTION="Component for managing Guided Tours functionality in Joomla 4"
+COM_GUIDEDTOURS_N_ITEMS_ARCHIVED="%s Items Archived"
+COM_GUIDEDTOURS_N_ITEMS_PUBLISHED="%s Items Published"
+COM_GUIDEDTOURS_N_ITEMS_UNPUBLISHED="%s Items Unpublished"
+COM_GUIDEDTOURS_N_ITEMS_TRASHED="%s Items Trashed"
+COM_GUIDEDTOURS_N_ITEMS_DELETED="%s Items Deleted"
+COM_GUIDEDTOURS_GUIDEDTOUR_STEPS_PAGE_TITLE="Guided Tours - List of Steps"
+COM_GUIDEDTOURS_STEP_FILTER_SEARCH_DESC="Search in Step name. Prefix with User: to search for a Step starting from User"
+COM_GUIDEDTOURS_FILTER_SEARCH_DESC="Search in Tour name. Prefix with User: to search for a Tour starting from User"
+COM_GUIDEDTOURS_OVERLAY_DESC="adds the CSS overlay effect to the Tour"
+COM_GUIDEDTOURS_EXTENSIONS_DESC="Will show the tour only for selected extensions. If used tours won't be displayed at other extensions."
+COM_GUIDEDTOURS_EXTENSIONS_LABEL="Component Selector"
+COM_GUIDEDTOURS_OVERLAY_LABEL="Overlay"
+COM_GUIDEDTOURS_URL_DESC="Enter the URL of the page from where you want to Start the Tour"
+COM_GUIDEDTOURS_URL_LABEL="URL"
+COM_GUIDEDTOURS_TOURID_LABEL="Tour ID"
+COM_GUIDEDTOURS_STEPID_LABEL="Step ID"
+COM_GUIDEDTOURS_TARGET_DESC="add the ClassName or ID of the element where you want to attach your Step Element"
+COM_GUIDEDTOURS_TARGET_LABEL="Target"
+COM_GUIDEDTOURS_POSITION_DESC="add the Position where you want to display your step. e.g. bottom, top, right, left"
+COM_GUIDEDTOURS_POSITION_LABEL="Position"
+JTOUR_STEPS ="Steps"
+JTOUR_DESCRIPTION="Description"
+JTOUR_TITLE="Tour Title"
+JSTEP_TITLE="Step Title"
+COM_GUIDEDTOURS_NO_NOTE=""
+COM_GUIDEDTOURS="Guided Tours"
+JSTEP_ASC="Step ascending"
+JSTEP_DESC="Step descending"
```

Comment:
all strings just for this component that do not appear elsewhere (you are not overriding) should have names beginning COM_GUIDEDTOURS...